doc/announcements-workflow.md

```diff
@@ -0,0 +1,59 @@
+# Announcements workflow
+
+An announcement is a marketing message delivered into the PIM through the communication panel. It can be: 
+- either the announcement of a new feature. In that case we call it an "update".
+- or any marketing message (not implemented yet)
+
+For now, only "updates" are pushed into the PIM, for CE and EE.
+
+## Where are written the content of the updates?
+
+The updates are written in this repository, in `content/updates` directory. There are multiple directories, corresponding to the month of the release of the feature.
+
+The syntax to write the content is the markdown.
+
+## Where are deployed the content of the updates?
+
+The updates are deployed:
+
+- on the helpcenter website. There is a dedicated page describing all the new features released the previous month. For example, `https://help.akeneo.com/pim/serenity/updates/2020-06.html`.
+- into the PIM, in the communication panel. The content of each announcement is the first paragraph to describe the feature.
+
+## When are deployed the content of the updates?
+
+The new features are released the 5th of each month, in the PIM and on the helpcenter website. For example, the new features of February 5th 2020 are released the March 5th 2020.
+
+The content of the directory `content/updates/2020-02` that is on `master` branch is not deployed in production before March 5th 2020.
+
+In staging, all announcements are deployed. The content of the directory `content/updates/2020-02` that is on `master` branch is deployed in staging, even if it's not yet March 5th 2020. It allows the Product Managers and the Product Marketing Managers to check the display of the new announcements before releasing it in production. 
+
+## How the updates are deployed?
+
+### On the helpcenter
+
+It generates the HTML content from the markdown. The content of the current month or future month is not generated if  the environment variable ONLY_PREVIOUS_MONTH_UPDATES equals `true`. Otherwise, it generates all updates.
+
+It deploys the helpcenter website on each PR merged on master.
+Also, in order to deploy automatically the 5th of the month the released features, there is a daily deployment of the helpcenter website.
+
+### On Firebase
+
+#### Push of the updates into firestore 
+
+It generates the JSON content from the markdown. The content of the current month or future month is not generated if  the environment variable ONLY_PREVIOUS_MONTH_UPDATES equals `true`. Otherwise, it generates all updates.
```

Comment:
```suggestion
It generates the JSON content from the markdown. The content of the current month or future month is not generated if the environment variable `ONLY_PREVIOUS_MONTH_UPDATES` equals `true`. Otherwise, it generates all updates.
```